### PR TITLE
Announce 4.50

### PR DESCRIPTION
**Pull request by fredemmott**

### CODE REVIEW

File: _posts/2020-03-24-hhvm-4.50.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "HHVM 4.50"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.50 is released! This release marks the end of support for 4.43;
+HHVM 4.45&ndash;4.49 remain supported, as do the 4.8 and 4.32 LTS releases.
```
Comment (jjergus) [inline _posts/2020-03-24-hhvm-4.50.markdown]: what happened to HHVM 4.44?

File: _posts/2020-03-24-hhvm-4.50.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Additionally, we are no longer building binary packages for MacOS High Sierra
+for any version of HHVM, as High Sierra is no longer offered by our build
+infrastructure provider..
```
Comment (jjergus) [inline _posts/2020-03-24-hhvm-4.50.markdown]: double period

File: _posts/2020-03-24-hhvm-4.50.markdown (hunk 1 above)
Comment (fredemmott) [reply _posts/2020-03-24-hhvm-4.50.markdown]: It was the oldest in last week’s announcement, and this leaves us with 6 concurrent releases; am I missing something?

Comment (jjergus) [reply _posts/2020-03-24-hhvm-4.50.markdown]: It says end of support for 4.43 and 4.45-... remain supported, so 4.44 was missing.

**Pull request merged**